tests/main/install-store-laaaarge/task.yaml

```diff
@@ -13,7 +13,17 @@ restore: |
     . "$TESTSLIB/systemd.sh"
 
     systemd_stop_units snapd.service snapd.socket
-    umount /tmp || true
```

Comment:
Could we do a `umount --lazy /tmp` here? This might be sufficient to fix the issue.

Reply:
BTW. I recall a note that `--lazy` does not work on 14.04 and you need to use `-l` instead.

Reply:
Sure, I'll make the change and test it.